### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by fredrikved**
Started with some refactoring.

### CODE REVIEW

File: workflows/export/asset_export-vx.go [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+
+	err = workflow.ExecuteActivity(ctx, activities.RcloneUploadDir, activities.RcloneUploadDirInput{
+		Source:      strings.Replace(outputFolder, utils.GetIsilonPrefix()+"/", "isilon:isilon/", 1),
+		Destination: fmt.Sprintf("s3prod:vod-asset-ingest-prod/" + ingestFolder),
+	}).Get(ctx, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	err = wfutils.PublishEvent(ctx, "asset.delivered", map[string]string{
```
Comment (andreasgangso) [inline workflows/export/asset_export-vx.go:202]: `github.com/bcc-code/bcc-media-platform/backend/events.AssetDelivered`

**Review by andreasgangso — CHANGES_REQUESTED** [review]
Kan hende git hadde maintaina history bedre her hvis man gjorde ting i 2 commits?
Se 1 kommentar på den ene eventen.

Ellers LGTM 👍

**Comment by fredrikved** [comment]
Will split into separate commits the next time I move things across packages.

File: utils/wfutils/files.go [hunk 2]
```diff
@@ -0,0 +1,78 @@
+package wfutils
```
Comment (KillerX) [inline utils/wfutils/files.go:1]: I keep reading this as `wtfutils` XD

File: utils/wfutils/files.go [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+		Path: path,
+	}).Get(ctx, nil)
+}
+
+func GetWorkflowOutputFolder(ctx workflow.Context) (string, error) {
+	info := workflow.GetInfo(ctx)
+
+	date := time.Now()
+
+	path := fmt.Sprintf("%s/%04d/%02d/%02d/%s", utils.GetIsilonPrefix()+"/Production/aux", date.Year(), date.Month(), date.Day(), info.OriginalRunID)
```
Comment (KillerX) [inline utils/wfutils/files.go:58]: Maybe we need the ability to switch out the `aux` part. Right now this is the correct place for many things but others are coming.

File: workflows/export/asset_export-vx.go
Comment (KillerX) [inline workflows/export/asset_export-vx.go:1]: Is there some logic for mixing `_` and `-` in file names?

**Review by KillerX — APPROVED** [review]

**Review by andreasgangso — APPROVED** [review]

**Pull request merged**